watchtower-plugin/src/net/http.rs

```diff
@@ -1,6 +1,7 @@
-use reqwest::Response;
+use reqwest::{Response,Client};
 use serde::{de::DeserializeOwned, Deserialize, Serialize};
 
+use log::debug;
```

Comment:
nit: Don't `use log:debug` as an import. Instead, call `log::debug!` when needed, both for consistency with the rest of the codebase but also for readability.

Reply:
ohh yes yess!